builder/postaggregation/post_aggregator.go

```diff
@@ -60,12 +61,19 @@ func Load(data []byte) (builder.PostAggregator, error) {
 		p = NewLongGreatest()
 	case "longLeast":
 		p = NewLongLeast()
+	case "quantileFromTDigestSketch":
+		p = NewQuantileFromTDigestSketch()
 	case "quantilesFromTDigestSketch":
 		p = NewQuantilesFromTDigestSketch()
 	case "quantilesDoublesSketchToQuantile":
 		p = NewQuantilesDoublesSketchToQuantile()
+	case "quantilesDoublesSketchToQuantiles":
+		p = NewQuantilesDoublesSketchToQuantiles()
+	case "quantilesDoublesSketchToHistogram":
+		p = NewQuantilesDoublesSketchToHistogram()
 	default:
 		return nil, errors.New("unsupported postaggregation type")
 	}
-	return p, json.Unmarshal(data, &p)
+	return p, json.Unmarshal(data,
```

Comment:
Does that pass the go lint ?

Reply:
yes it does, it's my IDE that formatted it differently. I'll fix it :)